packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -79,14 +78,12 @@ class ExpandableVariable extends StatelessWidget {
       shrinkWrap: true,
       dataDisplayProvider: (variable, onPressed) =>
           displayProvider(context, variable, onPressed, debuggerController),
-      onItemSelected: (variable) async =>
-          onItemPressed(variable, debuggerController),
+      onItemSelected: (variable) async => onItemPressed(variable),
```

Comment:
does onItemPressed need to be awaited or unawaited?

Reply:
Fyi @incendial. Wonder why the lints for unneeded async didn't trigger here.
This code could be further simplified to just 
`onItemSelected: onItemPressed`

Reply:
Agree, noted